plugins/widget/plugin.js

```diff
@@ -2901,6 +3004,80 @@
 		} );
 	}
 
+	function handleStylesOnDowncast( widget, widgetElement ) {
+		var styledElement;
+
+		if ( widgetElement ) {
```

Comment:
This could be simplified by reversing conditional and implementing early `return`.